downloads/swan-lake-release-notes/swan-lake-preview5/RELEASE_NOTE.md

```diff
@@ -0,0 +1,255 @@
+                        
+> `ballerina update`
+
+ Next, execute the command below to update to Swan Lake Preview 5.
+
+ > `ballerina dist pull slp5b1`                 
+
+However, if you are using a Ballerina version below 1.1.0, install via the [installers](https://ballerina.io/downloads/).
+
+#### For new users
+
+If you have not installed Ballerina, then download the [installers](https://ballerina.io/downloads/) to install.
+
+### What is new in Ballerina Swan Lake Preview 5
+
+#### Language
+
+##### Isolated objects
+
+A class definition or an object-type descriptor can now be marked as `isolated`. All fields of an `isolated` object that are not `final` or are not a subtype of `readonly` or `isolated object {}` have to be `private` fields.
+
+The methods of an `isolated` object that access such fields can only refer to `self` within a `lock` statement, which ensures that mutable state is only accessed within `lock` statements. If a method accesses such a field, additional rules apply to how values are transferred in and out in order to ensure that there are no references into the object’s mutable storage except via isolated objects. The initial value expressions of the fields also need to ensure the same.
+
+```ballerina
+type Coordinates record {|
+    int latitude;
+    int longitude;
+|};
+
+isolated class Hotels {
+
+    private map<Coordinates> hotels = {};
+
+    isolated function add(string name, Coordinates coordinates) {
+        lock {
+            self.hotels[name] = coordinates.clone();
+        }
+    }
+
+    isolated function getCoordinates() returns map<Coordinates> {
+        lock {
+            return self.hotels.clone();
+        }
+    }
+}
+```
+
+An `isolated` function can also access a `final` variable if its type is a subtype of `isolated object {}`.
+
+```ballerina
+final Hotels hotels = new;
+
+isolated function getCoordinates() returns Coordinates[] => hotels.getCoordinates().toArray();
+```
+
+##### Removed XML functional constructors
+
+XML functional constructor syntax has been removed. In place of XML functional constructors, `'xml:createXXX` functions should be used. 
+
+**Disallowed syntax**
+```ballerina
+'xml:Element element = 'xml:Element("elem");
+'xml:Comment comment = 'xml:Comment("comment content");
+'xml:ProcessingInstruction proceInstruc = 'xml:ProcessingInstruction("DONOT", "print this");
+'xml:Text text = 'xml:Text("this is a character sequence");
+```
+
+**Valid syntax**
+```ballerina
+'xml:Element element = 'xml:createElement("elem");
+'xml:Comment comment = 'xml:createComment("comment content");
+'xml:ProcessingInstruction proceInstruc = 'xml:createProcessingInstruction("DONOT", "print this");
+'xml:Text text = 'xml:createText("this is a character sequence");
+```
+
+##### Unified object method scope and object field scope 
+
+Object field scope and object method scope are unified into a single object scope. This means that it's no longer possible to have fields and methods with the same name.
```

Comment:
are unified or have been unified? :thinking:

Reply:
I was also wondering the same. We seem to have used a mix across the release note, though.

Reply:
@MaryamZi

Reply:
"have been" sounds better. Will change.